docs/openhabian.md

```diff
@@ -275,7 +274,7 @@ During and after the first boot of your Raspberry Pi, the green on-board LED wil
 -->
 
 **RPi note:**
-The progress indication via the **green Raspberry Pi LED** is currently not possible and hence not part of the openHABian v1.3+ image.
+The progress indication via the **green Raspberry Pi LED** is currently not possible.
```

Comment:
no there's noticket, I have never seen this function either (it's one of those legacy things Thomas added to the docs long ago and noone dares throwing out things when you have not seen them before)

Reply:
Same here, never seen it, on the other hand I doubt it is going to change any time soon. I don't think it would hurt to remove it. On the other hand it is not hurting us to leave it there.

Reply:
@mstormi you have my support for removing it. I have not seen the code which tries to control the LEDs...

Reply:
ok, I removed the docs on indicator LED